lib/helper/REST.js

```diff
@@ -219,5 +219,187 @@ class REST extends Helper {
 
     return this._executeRequest(request);
   }
+
+
+  /**
+ * Checks if the response code is equal to what was provided
+ *
+ * ```js
+ * I.seeResponseCodeIs(200);
+ * ```
+ *
+ * @param {string|number} statusCode
+ */
+  seeResponseCodeIs(statusCode) {
+    expect(this.response.status).to.equal(statusCode, `Status code ${this.response.status} of response is not equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is not equal to what was provided
+   *
+   * ```js
+   * I.dontSeeResponseCodeIs(200);
+   * ```
+   *
+   * @param {string|number} statusCode
+   */
+  dontSeeResponseCodeIs(statusCode) {
+    expect(this.response.status).to.not.equal(statusCode, `Status code ${this.response.status} of response is equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is an information response code by checking if its value is between 100-199
+   *
+   * ```js
+   * I.seeResponseWasInformation();
+   * ```
+   */
+  seeResponseWasInformation() {
```

Comment:
I don't think this method will be valuable. 

Maybe better to have something like `seeResponseCodeBetween(min,max)`

I don't know lot of people who would check for 1xx keys. Really.

Reply:
If you are using Codecept as a REST API testing method only you and the API you are building outputs a 1xx status code then you would probably want a readable method to test it. Seems like I am in the minority here so I will remove that method and also add the responseCodeBetween.

I can't think of a reason to assert between a specific range rather than just asserting on an exact status code but ill add the method anyway